app/models/profile_field.rb

```diff
@@ -19,6 +19,7 @@ class ProfileField < ApplicationRecord
   validates :label, presence: true, uniqueness: { case_sensitive: false }
   validates :attribute_name, presence: true, on: :update
   validates :show_in_onboarding, inclusion: { in: [true, false] }
+  validates :group, presence: true
```

Comment:
@citizen428 want to double check with you, but I think that a profile field needs to always be entered.

Reply:
oops I meant a group

Reply:
So, theoretically, it can be empty:

```ruby
 t.string "group"
```

BUT: since we already know that we want to make groups a separate model soon, this should be ok.